appinventor/appengine/src/com/google/appinventor/client/TopToolbar.java

```diff
@@ -103,12 +100,6 @@
   private static final String WIDGET_NAME_IMPORTTEMPLATE = "ImportTemplate";
   private static final String WIDGET_NAME_EXPORTALLPROJECTS = "ExportAllProjects";
   private static final String WIDGET_NAME_EXPORTPROJECT = "ExportProject";
-  private static final String WIDGET_NAME_COMPONENTS = "Components";
```

Comment:
I think it's okay to get rid of these since we aren't using them at present (and associated `Action`s below). @jisqyv Do you have any objection?